docs/installation/presto-connector-installation.rst

```diff
@@ -0,0 +1,40 @@
+
+==================
+Adding a Connector
+==================
+
+In Presto, connectors allow you to access different data sources -- e.g.,
+Hive, PostgreSQL, or MySQL. In order to add a connector, you need to create a
+configuration file, and then distribute it to the cluster via ``presto-admin``.
+
+For example, in order to add the Hive connector, create a file ``hive.properties`` in ``/etc/opt/prestoadmin/connectors``.
+
+For Cloudera CDH 5, it should have the following content: ::
+
+    connector.name=hive-cdh5
+    hive.metastore.uri=thrift://<metastore-host-or-ip>:<metastore-port>
+
+
+For Hadoop 2.0+ (including HDP), it should have the following content: ::
+
+   connector.name=hive-hadoop2
+   hive.metastore.uri=thrift://<metastore-host-or-ip>:<metastore-port>
+
+
+There are additional properties and possible configurations in the
+`Hive connector documentation <https://prestodb.io/docs/current/connector/hive.html>`_,
+for example if you have a HA Hadoop cluster.
+
+After adding the connector file, you need to add the connector via ``presto-admin``: ::
+
+    sudo ./presto-admin connector add hive
+
+
+In order to query using the newly added connector, you need to restart the
```

Comment:
Propose: "Once the new connector configuration has been distributed,  :ref:`restart <reference to server restart>` Presto."